app/views/devise/shared/_links.html.haml

```diff
@@ -0,0 +1,10 @@
+- if controller_name != 'sessions'
+  = link_to "Log in", new_session_path(resource_name)
```

Comment:
please remove this file, no need to have it in the code base

Reply:
Don't we need it for loggging in?

Reply:
nope same here, try to run the tests without this file

Reply:
The sign up option is linked to this, so if this is removed, the login function remains but sign up option is removed as its bound to this.